markdown/org/docs/patterns/diana/instructions/en.md

```diff
@@ -1,63 +1,67 @@
 
-## Finish the back neckline
+### Step 1: Finish the back neckline
 
-Cut a strip of self fabric the length of your back neckline, and around 3 cm wide.  
-Press a fold into this strip, 1 cm from the edge.
+- Cut a strip of main fabric the length of your back neckline, and around 3 cm wide.
+- Press a fold into this strip, 1 cm from the edge.
 
 <Tip>
 
-Make this a bias cut strip if you are using a woven fabric, 
-or a strip with the stretch direction running along the long edge if you are using a knit. 
+For woven fabric make this a bias cut strip.  
+For knit fabrc Make a strip with the stretch of the fabric running along the long edge.
 
 </Tip>
 
-Align the non-folded edge of the strip with the back neckline with good sides together. 
-Sew the strip to the back neckline and trim the seam. Turn the strip to the wrong side of the fabric
-and topstitch along the folded edge to keep it in place.
+- With *good sides together* align the non-folded edge of the strip with the back neckline.
+- Sew the strip to the back neckline and trim the seam. 
+- Turn the strip to the wrong side of the fabric and *topstitch* along the folded edge to keep it in place.
 
-## Close shoulder seams
+### Step 2: Close shoulder seams
 
 ![This drawing was too nice not to use](neckline.jpg)
 
-Finish the raw edge on the front neckline in a way that suits your fabric 
-(if it doesn’t fray, you can leave it unfinished). 
+- Finish the raw edge on the front neckline in a way that suits your fabric (if it doesn’t fray, you can leave it unfinished).
+- With *good sides together*, place the front on the back along the shoulder seams.
```

Comment:
Small suggestion: I don't find "along the shoulder seams" clearer than "aligning the shoulder seams".

Reply:
Yea I can see that one sec.

Reply:
🙏